### PR TITLE
pkg/{porter, cnab},cmd,docs: remove the capability to pass a file path to the -c and -p flags

### LINKED ISSUE #1809: Can no longer provide kubeconfig as credential to a bundle on installation
## Describe the bug
Since Porter v1.0.0-alpha.4, I am no longer able to provide a kubeconfig file as a credential to a bundle during installation.

## To Reproduce
1. Create a file `kubeconfig.json` with the following content (substituting the path to a valid kubeconfig file for a suitable test environment)
```json
{
  "credentials": [
    {
      "name": "kubeconfig",
      "source": {
        "path": "/tmp/test/kubeconfig.yml"
      }
    }
  ]
}
```
2. Create a file `parameters.json` with the following content:
```json
{
  "parameters": [
    {
      "name": "registry",
      "source": {
        "value": "test-registry"
      }
    },
    {
      "name": "username",
      "source": {
        "value": "test-user"
      }
    },
    {
      "name": "password",
      "source": {
        "value": "test-pass"
      }
    },
    {
      "name": "namespace",
      "source": {
        "value": "default"
      }
    }
  ]
}
```
3. Run this porter command: `porter install ips --reference registry.centros.cloud/library/image-pull-secret:v1.0.1 -c ./kubeconfig.json -p ./params.json`

4. With Porter v1.0.0.alpha3, this command completes successfully:
```
vmuser@sandbox:~/test/porter$ porter --version
porter v1.0.0-alpha.3 (09840b4c)

vmuser@sandbox:~/test/porter$ porter install ips --reference registry.centros.cloud/library/image-pull-secret:v1.0.1 -c ./kubeconfig.json -p ./params.json
installing ips...
executing install action from image-pull-secret (installation: ips)
Prepare manifest

apiVersion: v1
kind: Secret
metadata:
  name: ips
  labels:
  namespace: default
type: kubernetes.io/dockerconfigjson
data:
  .dockerconfigjson: eyJhdXRocyI6eyJ0ZXN0LXJlZ2lzdHJ5Ijp7InVzZXJuYW1lIjoidGVzdC11c2VyIiwicGFzc3dvcmQiOiJ0ZXN0LXBhc3MiLCJlbWFpbCI6IiIsImF1dGgiOiJkR1Z6ZEMxMWMyVnlPblJsYzNRdGNHRnpjdz09In19fQ==
Create image pull secret
secret/ips configured
execution completed successfully!

vmuser@sandbox:~/test/porter$ kubectl get secret/ips
NAME   TYPE                             DATA   AGE
ips    kubernetes.io/dockerconfigjson   1      3s
```

5. With v1.0.0.alpha.4, in an otherwise completely identical environment, an error occurs (also tested with alpha.5; behaviour still persists):
```
vmuser@sandbox:~/test/porter$ porter --version
porter v1.0.0-alpha.4 (4b2126c0)

vmuser@sandbox:~/test/porter$ porter install ips --reference registry.centros.cloud/library/image-pull-secret:v1.0.1 -c ./kubeconfig.json -p ./params.json
installing ips...
Error: could not load credentials: bundle requires credential for kubeconfig

```

## Expected behavior
Porter should still succeed at installing the bundle

## Porter Command and Output
See above; additionally, this is the output of the command above when adding the `--debug` parameter, in case that helps identify the issue:
```
detecting Porter config in /home/vmuser/.porter
loaded Porter config from 
Using '' namespace from configuration
Pulling bundle registry.centros.cloud/library/image-pull-secret:v1.0.1
waiting for the mongo service to be ready
Connecting to mongo database porter at [localhost:27018]
resolving parameters for installation /ips
No parameter sources defined, skipping
Pulling bundle registry.centros.cloud/library/image-pull-secret:v1.0.1
resolving parameters for installation /ips
No parameter sources defined, skipping
installing ips...
{"credentials":{"kubeconfig":{"path":"/root/.kube/config","required":true}},"custom":{"sh.porter":{"commit":"3018b91c","manifest":"LS0tCm5hbWU6IGltYWdlLXB1bGwtc2VjcmV0CnZlcnNpb246IDEuMC4xCmRlc2NyaXB0aW9uOiBBIHBvcnRlciBidW5kbGUgZm9yIGNyZWF0aW5nIGs4cyBpbWFnZSBwdWxsIHNlY3JldHMKcmVnaXN0cnk6IHJlZ2lzdHJ5LmNlbnRyb3MuY2xvdWQvbGlicmFyeQpkb2NrZXJmaWxlOiBEb2NrZXJmaWxlLnRtcGwKbWl4aW5zOgogIC0gZXhlYwogIC0ga3ViZXJuZXRlcwpwYXJhbWV0ZXJzOgogIC0gbmFtZTogdXNlcm5hbWUKICAgIHR5cGU6IHN0cmluZwogIC0gbmFtZTogcGFzc3dvcmQKICAgIHR5cGU6IHN0cmluZwogIC0gbmFtZTogcmVnaXN0cnkKICAgIHR5cGU6IHN0cmluZwogIC0gbmFtZTogZW1haWwKICAgIHR5cGU6IHN0cmluZwogICAgZGVmYXVsdDogIiIKICAtIG5hbWU6IG5hbWVzcGFjZQogICAgdHlwZTogc3RyaW5nCiAgICBkZWZhdWx0OiBkZWZhdWx0CmNyZWRlbnRpYWxzOgogIC0gbmFtZToga3ViZWNvbmZpZwogICAgcGF0aDogL3Jvb3QvLmt1YmUvY29uZmlnCmluc3RhbGw6CiAgLSBleGVjOgogICAgICBkZXNjcmlwdGlvbjogUHJlcGFyZSBtYW5pZmVzdAogICAgICBjb21tYW5kOiAiLi9oZWxwZXJzLnNoIgogICAgICBhcmd1bWVudHM6CiAgICAgICAgLSBwcmVwYXJlLW1hbmlmZXN0CiAgICAgICAgLSAiXCJ7eyBpbnN0YWxsYXRpb24ubmFtZSB9fVwiIgogIC0ga3ViZXJuZXRlczoKICAgICAgZGVzY3JpcHRpb246IENyZWF0ZSBpbWFnZSBwdWxsIHNlY3JldAogICAgICBtYW5pZmVzdHM6CiAgICAgICAgLSBrOHMvaXBzLnlhbWwKICAgICAgd2FpdDogdHJ1ZQp1cGdyYWRlOgogIC0gZXhlYzoKICAgICAgZGVzY3JpcHRpb246IFByZXBhcmUgbWFuaWZlc3QKICAgICAgY29tbWFuZDogIi4vaGVscGVycy5zaCIKICAgICAgYXJndW1lbnRzOgogICAgICAgIC0gcHJlcGFyZS1tYW5pZmVzdAogICAgICAgIC0gIlwie3sgaW5zdGFsbGF0aW9uLm5hbWUgfX1cIiIKICAtIGt1YmVybmV0ZXM6CiAgICAgIGRlc2NyaXB0aW9uOiBVcGRhdGUgaW1hZ2UgcHVsbCBzZWNyZXQKICAgICAgbWFuaWZlc3RzOgogICAgICAgIC0gazhzL2lwcy55YW1sCiAgICAgIHdhaXQ6IHRydWUKdW5pbnN0YWxsOgogIC0gZXhlYzoKICAgICAgZGVzY3JpcHRpb246IERlbGV0ZSBpbWFnZSBwdWxsIHNlY3JldAogICAgICBjb21tYW5kOiAiLi9oZWxwZXJzLnNoIgogICAgICBhcmd1bWVudHM6CiAgICAgICAgLSBkZWxldGUtc2VjcmV0Cg==","manifestDigest":"e59e92fbd95df45c70cdded6617a33aa95789266e2d442e9c8f82d7b6f055e0f","mixins":{"exec":{},"kubernetes":{}},"version":"v0.38.1"},"sh.porter.file-parameters":{}},"definitions":{"email-parameter":{"default":"","type":"string"},"namespace-parameter":{"default":"default","type":"string"},"password-parameter":{"type":"string"},"porter-debug-parameter":{"$comment":"porter-internal","$id":"https://porter.sh/generated-bundle/#porter-debug","default":false,"description":"Print debug information from Porter when executing the bundle","type":"boolean"},"registry-parameter":{"type":"string"},"username-parameter":{"type":"string"}},"description":"A porter bundle for creating k8s image pull secrets","invocationImages":[{"contentDigest":"sha256:d2c42cc57e1c2b28dfdb1c084b2bb6dfb5a5bd573c00666f7bffd4f92e1e1a08","image":"registry.centros.cloud/library/image-pull-secret-installer:v1.0.1","imageType":"docker","mediaType":"application/vnd.docker.distribution.manifest.v2+json","size":2003}],"name":"image-pull-secret","parameters":{"email":{"definition":"email-parameter","destination":{"env":"EMAIL"}},"namespace":{"definition":"namespace-parameter","destination":{"env":"NAMESPACE"}},"password":{"definition":"password-parameter","destination":{"env":"PASSWORD"},"required":true},"porter-debug":{"definition":"porter-debug-parameter","description":"Print debug information from Porter when executing the bundle","destination":{"env":"PORTER_DEBUG"}},"registry":{"definition":"registry-parameter","destination":{"env":"REGISTRY"},"required":true},"username":{"definition":"username-parameter","destination":{"env":"USERNAME"},"required":true}},"requiredExtensions":["sh.porter.file-parameters"],"schemaVersion":"v1.0.0","version":"1.0.1"}
Error: could not load credentials: bundle requires credential for kubeconfig
``` 

## Version
See above.

### PR DESCRIPTION
**Pull request by VinozzZ**
Signed-off-by: Yingrong Zhao <yingrong.zhao@gmail.com>

# What does this change

Currently, when running an action with a bundle, users can pass in parameter sets and credential sets through a file from command line flags `-c` and `-p`. This functionality was introduced originally to allow scripting capability with porter. Now porter has dedicated credential and parameter management command, we can safely remove this capability to allow more clear user flow and reduce code complexity

# What issue does it fix
Closes #1809 

[1]: https://porter.sh/src/CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer

# Checklist
- [ ] Did you write tests?
- [ ] Did you write documentation?
- [ ] Did you change porter.yaml or a storage document record? Update the corresponding schema file.
- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

# Reviewer Checklist
* Comment with /azp run test-porter-release if a magefile or build script was modified
* Comment with /azp run porter-integration if it's a non-trivial PR

[contributors]: https://porter.sh/src/CONTRIBUTORS.md

### CODE REVIEW

File: pkg/cnab/provider/credentials.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (r *Runtime) loadCredentials(b cnab.ExtendedBundle, args ActionArguments) (
 		var cset credentials.CredentialSet
 		var err error
 		if r.isPathy(name) {
-			cset, err = r.loadCredentialFromFile(name)
+			return nil, errors.Errorf("cannot use file path %s as credential set source", name)
```
Comment (carolynvs) [inline pkg/cnab/provider/credentials.go:29]: What if we remove this check (and isPathy) entirely? Then if someone tries to pass a file, the error will be from when we tried to load the credential set, e.g. "credential set ./mycreds.json not found".

As we have been making breaking changes for v1, I have been removing deprecation warnings, shims and checks for old behavior that was only supported in v0.* versions. The v1 release is our one chance to make a correct previous decisions and simplify the code, before we _have_ to keep around a bunch of code for backwards compatibility. 😀

Comment (VinozzZ) [reply pkg/cnab/provider/credentials.go:29]: Sounds good. That makes sense, Thanks for explaining the context with me!

Comment (carolynvs) [reply pkg/cnab/provider/credentials.go]: Would you please make the same change for parameters as well so they are consistent?

**Comment by carolynvs** [comment]
Looks like there are still some integration tests that are using this feature and need to be updated to use the apply command first:

**Comment by carolynvs** [comment]
@VinozzZ On your local machine, please run `mage build` and then check in the changes to the files in the docs directory. We have generated docs for our command line help text that need to be refreshed after you made changes to the command help text. Then this will be ready to merge! 🚀

**Review by carolynvs — APPROVED** [review]
:shipit:

**Pull request merged**